user/Lambda.py

```diff
@@ -67,12 +55,30 @@ def Lambda_Acquire(acquire_time=0.1, acquire_period=0.11,num_images=100,file_nam
     @bpp.stage_decorator([scaler1])
     @bpp.monitor_during_decorator(monitored_things)
     def inner():
+        # write metadata to the dm_pars (various registers)
+        yield from bps.mv(
+            dm_pars.root_folder, file_path,
+            dm_pars.parent_folder, os.path.dirname(file_path),
+            dm_pars.data_folder, file_name,
+            dm_pars.datafilename, areadet.get_plugin_file_name(),
+            # TODO: what else?
```

Comment:
start_time, end_time to 8idi:StrReg6 and 7, after the acq, ring_i_beg and _end to 8idi:Reg121 and 122
![registers](https://user-images.githubusercontent.com/48140482/64460071-1881c700-d0bf-11e9-95f6-75157af06567.png)
![registers1](https://user-images.githubusercontent.com/48140482/64460115-36e7c280-d0bf-11e9-93e7-1483e229e489.png)

Reply:
done but needs more discussion